docs/index.md

```diff
@@ -0,0 +1,85 @@
+
+## Why the FVM is useful
+
+There has been a huge demand from the developer community for compatibility with Ethereum and Solidity out of the box. With the FVM, developers can access the massive corpus of audited and battle-tested smart contracts written in Solidity for the Ethereum Virtual Machnine (EVM). The FVM also supports other foreign runtimes and virtual machines, such as JS/SES, LLVm, IR, and BPF.
+
+The FVM is flexible. Because it's not tightly-coupled to Filecoin, developers can adapt to it in other contexts, such as the Interplanetary File System (IPFS) and Interplanetary Linked Data (IPLD).
+
+The FVM is easy for developers who know IPLD to adapt to. The state tree and actor states that the FVM uses to guide storage deals from pre-commit through committed states are IPLD objects.
+
+## Vision and goals
+
+To mitigate the well-known risks of compiling Solidity used by the EVM, we're not working directly in Solidity. Instead, we're building the reference SDK in Rust, because it produces very succinct WASM code and keeps runtime costs down.
+
+Because the FVM essentially executes computation on IPLD inputs to produce outputs, building libraries and primitives to interact with IPLD data is a core priority.
+
+Because user-defined actors will exponentially increase demand for space on the blockchain, we're looking into solutions, such as sharding (a kind of partitioning) and hierarchical consensus with parallel execution, both of which support speedier processing.
+
+## Use-cases
+
+Here are some use cases that are possible with the FVM. With centralized data centers, a lot of these use cases were incredibly hard, if not impossible. With decentralization and Layer 1 programmability (computation for state changes), we're unleashing boundless opportunities.
+
+### Decentralized, verifiable computation
+
+The FVM enables both computation and storage under a single roof. Imagine being able to apply a computation on data, incentivizing its execution all the way to the end, and certifying that the result AND the output are verifiably correct.
+
+Imagine being able to link all of that to the Dataverse (for sharing, preserving, citing, exploring, and analyzing data) and tokenization of datasets.
+
+### Dataverse and Data DAOs
+
+The Dataverse is an open source software project for sharing, preserving, citing, exploring, and analyzing research data. See [The Dataverse Project](https://dataverse.org/).
+
+DAOs are Data Access Objects, patterns that provide an abstract interface to a database or other persistence mechanism. They provide specific data operations without exposing details of the database. See [Data access object](https://en.wikipedia.org/wiki/Data_access_object).
```

Comment:
Wrong kind of DAO. In blockchain land, a DAO is a Decentralized Autonomous Organization. https://en.wikipedia.org/wiki/Decentralized_autonomous_organization

Reply:
@johnnymatthews I know about DAOs for blockchain, but I thought we were talking about data here. Decentralized Autonomous Organization doesn't seem to make sense here.  Are you sure?

Reply:
Will work on the intro paragraph, in response to [slack convo](https://filecoinproject.slack.com/archives/C029MT4PQB1/p1655496096856579), and on this use-case section.